### ISSUE TITLE
[HL1] "Hazard Course" labelled "Training Room" in the menu

### ISSUE THREAD

**Issue by joewintergreen** [ValveSoftware/halflife#2172]
When Steam came out in 2003, HL1's [awesome old menu](https://twitter.com/joewintergreen/status/676353781257277440)  was replaced with a kind of lame new one, which incorrectly labels the Hazard Course as the "Training Room", a name which is apparently an HL2 relic. As HL1's finest level, the Hazard Course deserves better.

**Comment by Sockman1** [comment]
To add to this, they should bring back the old menu if possible, or at least the style of the old menu.

**Comment by joewintergreen** [comment]
Yeah, Xash does a great job bringing the old menu back minus the issues it had (forcing display res to 640x480, etc). I imagine there are technical reasons why it would be a headache to keep both that menu and the new windowed console, etc.
